docs/vectorgrid-api-docs.html

```diff
@@ -385,6 +385,29 @@ <h3 id='symbolizer-method'>Methods</h3>
                 fillOpacity: 0
             }
         },
+        // A function for styling features dynamically, depending on their
+        // properties, the map&#39;s zoom level, and the layer's geometry type
+        // (point, line, polygon)
+        water: function(properties, zoom, type) {
```

Comment:
Happy to change `type` to another argument name. as it's not 100% clear what it means. However, I don't know if there;s a naming convention I should follow. `geom_type`? `geometry_type`? `geometry`?

Reply:
I'd personally go for `geometry_dimension`, just because of the similarity with http://postgis.net/docs/ST_Dimension.html